curriculum/challenges/english/blocks/review-dictionaries-and-sets/67f39babe1e2ec1fb6eea32a.md

```diff
@@ -242,10 +242,10 @@ my_set.clear()
 
 ```python
 my_set = {1, 2, 3, 4, 5} 
-your_set = {2, 3, 4, 5}
+your_set = {2, 3, 4, 5, 6}
```

Comment:
sorry,the changes to the examples are a bit more complex, we want an example that gives `True` for `issubset`  and `issuperset`, so maybe each example should have its own sets defined

Reply:
that would make the examples work, but I still wonder if there is a specific reason to have the `issubset` and `issuperset` example return `True` instead of `False`